app/views/admin/businesses/hiring_invoice_requests/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render Admin::ContainerComponent.new do %>
+  <%= render Admin::ContentComponent.new do %>
+    <%= render Admin::DescriptionListComponent.new(t(".title")) do |list| %>
+      <% list.with_description "Full name" do %>
```

Comment:
Please move this string to the locale file.

Reply:
added in e482d5cf1d386557203904931dfe19ffc0356277